airbyte-integrations/connectors/source-hubspot/source_hubspot/api.py

```diff
@@ -451,7 +452,7 @@ def read(self, getter: Callable, params: Mapping[str, Any] = None) -> Iterator:
                 self._start_date = self._state
 
     def read_chunked(
-        self, getter: Callable, params: Mapping[str, Any] = None, chunk_size: pendulum.Interval = pendulum.interval(days=1)
```

Comment:
Because of pendulum version is outdated, and pendulum.Interval is no longer used (deprecated), instead we should use pendulum.duration, which provides us the same functionality.